utah-design-system-website/src/react/components/websiteContent/Search.jsx

```diff
@@ -19,9 +20,56 @@ export function Search() {
     if (param) { setQuery(param); }
   };
 
+  const checkSkipLink = () => {
+    const link = document.getElementById('skip-link');
+    if (link) {
+      setSkipLink(link);
+    } else {
+      // Create observer to watch for new nodes
+      const obs = new MutationObserver(() => {
+        const link = document.getElementById('skip-link');
+        if (link) {
+          setSkipLink(link);
+        }
+      });
+      // @ts-ignore
+      if (!observer) setObserver(obs);
+    }
+  };
+
+  const setSkipLink = (/** @type {HTMLElement} */ link) => {
+    // Update skip link
+    link.onclick = (e) => {
+      e.preventDefault();
+      // Focus on search input
+      const content = document.getElementById('searchInput');
+      content?.focus();
+    };
+    // Disconnect observer when done
+    // @ts-ignore
+    if (observer) observer.disconnect();
+  };
+
+  useEffect(() => {
+    if (observer) {
+      // @ts-ignore
+      observer.observe(document, { subtree: true, childList: true });
+    }
+  }, [observer]);
+
   useEffect(() => {
     insertScript();
     getSearchTerm();
+    checkSkipLink();
+  }, []);
+
+  useEffect(() => () => {
+    // Disconnect observer on unmount
+    // @ts-ignore
+    if (observer) observer.disconnect();
+    // Reset link behavior
+    const link = document.getElementById('skip-link');
+    if (link) link.onclick = () => null;
   }, []);
 
   return (
```

Comment:
on line 81, it has `“enter”`. Most the rest of the site uses `&quot;` for quotations to avoid mixed quotes used throughout the site and to be more html-esque.